### PR TITLE
Fix broken links in Security Service integrations packages

### PR DESCRIPTION
**Pull request by alaudazzi**
This PR fixes several broken links, as per the following list:

/packages/wiz @elastic/security-service-integrations
https://integrate.wiz.io/reference/prerequisites
=> remove the link as the referenced content is not publicly available

https://integrate.wiz.io/reference/vulnerability-finding
=> remove the link as the referenced content is not publicly available

~/packages/checkpoint @elastic/security-service-integrations
http://downloads.checkpoint.com/dc/download.htm?ID=108670
=> https://sc1.checkpoint.com/documents/R81/WebAdminGuides/EN/CP_R81_SecurityManagement_AdminGuide/Topics-SECMG/Welcome.htm~

/packages/bbot @elastic/security-service-integrations
https://www.blacklanternsecurity.com/bbot/scanning/
=> https://www.blacklanternsecurity.com/bbot/Stable/scanning/

/packages/cisco_duo @elastic/security-service-integrations
https://admin.duosecurity.com/
=> https://admin.duosecurity.com/login

/packages/ti_cybersixgill @elastic/security-service-integrations
https://www.cybersixgill.com/products/darkfeed/
=> https://cybersixgill.com/products/cyber-threat-intelligence/darkfeed

/packages/google_workspace @elastic/security-service-integrations
https://developers.google.com/admin-sdk/reports
=> https://developers.google.com/admin-sdk/reports/reference/rest

/packages/lumos @elastic/security-service-integrations
https://api.lumos.com/activity_logs
=> remove the link as the referenced content is not publicly available

/packages/tychon @elastic/security-service-integrations
https://tychon.io/start-a-free-trial/
=> https://tychon.io/request-a-pilot/

/packages/thycotic_ss @elastic/security-service-integrations
https://docs.thycotic.com/secrets/current/events-and-alerts/secure-syslog-cef
=> https://docs.delinea.com/online-help/secret-server/alerts-events/logs/secure-syslog-cef/index.htm?Highlight=secure-syslog-cef

/packages/authentik @elastic/security-service-integrations
https://docs.goauthentik.io/developer-docs/api/reference/events-events-list
=> https://docs.goauthentik.io/docs/developer-docs/api/reference/events-events-list

https://docs.goauthentik.io/developer-docs/api/reference/core-groups-list
=> https://docs.goauthentik.io/docs/developer-docs/api/reference/core-groups-list

https://docs.goauthentik.io/developer-docs/api/reference/core-users-list
=> https://docs.goauthentik.io/docs/developer-docs/api/reference/core-users-list

## Related issue

https://github.com/elastic/integration-docs/issues/585

### CODE REVIEW

File: packages/wiz/_dev/build/docs/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ This module has been tested against the **Wiz API Version v1**.
 6. Save the integration.
 
 **Note:**
-  - Vulnerability data_stream pulls vulnerabilities from the previous day. For more information, refer to the link [here](https://integrate.wiz.io/reference/vulnerability-finding)
+  - Vulnerability data_stream pulls vulnerabilities from the previous day. For more information, refer to the link [here](https://www.wiz.io/integrations?category=vulnerability_response_management#explore-catalog)
```
Comment (kcreddy) [inline packages/wiz/_dev/build/docs/README.md:76]: Unrelated link. I think the link should point to the vulnerability API endpoint, which doesn't seem to be publicly available. Better remove the link entirely here.

**Review by kcreddy — COMMENTED** [review]
For each integration, add `changelog.yml` entry and update `mainfest.yml` like earlier PR: https://github.com/elastic/integrations/pull/11926/files

**Comment by kcreddy** [comment]
I also don't see `checkpoint` package in the list of changes, but is in the PR description for fix.

**Comment by taylor-swanson** [comment]
> I also don't see `checkpoint` package in the list of changes, but is in the PR description as ones to be fixed.

`checkpoint` is owned by @elastic/sec-deployment-and-devices, so it should be handled in a separate PR with the other deployment and devices integrations.

File: packages/wiz/_dev/build/docs/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Wiz continuously prioritizes critical risks based on a deep cloud analysis acros
 
 The Wiz integration collects four types of data: Audit, Cloud Configuration Finding, Issue and Vulnerability.
 
-Reference for [Graph APIs](https://integrate.wiz.io/reference/prerequisites) of Wiz.
+Reference for [Graph APIs](https://wiz.readthedocs.io/en/latest/api_reference/graph.html) of Wiz.
```
[the remark this answers is not in the document]
Comment (alaudazzi) [reply packages/wiz/_dev/build/docs/README.md]: If the referenced content is not publicly available, I suggest we remove the link.

**Review by kcreddy — APPROVED** [review]
LGTM 👍🏼 

CI error is due to mismatch between `_dev/build/docs/README.md` and `docs/README.md` by running `elastic-package build`. 

Please also change the PR description with updated new links.

**Pull request merged**